components/Person.js

```diff
@@ -78,16 +86,6 @@ export default function Person(props) {
             .avatar_container {
               width: 100%;
             }
-            .person_info {
```

Comment:
The way the code is currently organized makes it hard to maintain a component's behavior
```css
.person_info {
}
......
@media screen and (min-width: 768px) {
    .person_info {
    }
}
```
This means that the CSS for the same element is split in 2 separate chunks, that interact with each other when it's rendered. The better way is to group the media queries into the element CSS like below
```css
.person_info {
    ....
    @media screen and (min-width: 768px) {
    }
}
```
However, `style-jsx` doesnt support this standard CSS syntax. So we decide to move to `styled-components` instead